controllers/list.controller.js

```diff
@@ -0,0 +1,15 @@
+const { list } = require("../models");
+
+exports.add = (req, res) => {
+  if (!req.body["name"].trim()) {
```

Comment:
you also need to check:

1. If date's valid
2. If expense text is valid (not empty or smt.)

Also you need error handling for some cases: for example - method name is invalid, there's no related table in db and other.